.idea/gradle.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you don't needs this file to be on git. Remove it, probably by either finding 'stop tracking' or 'add to ignore list'